### ISSUE TITLE
Support the latest Android Studio (Giraffe)

### ISSUE THREAD

**Issue by jefft0** [berty/berty#4753]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Berty product

Mobile app

### Feature request

Most people new to the project will install the latest Android Studio (Giraffe). Opening the project gives the error below, some kind of incompatibility of the Gradle version.

Unsupported Java. 
Your build is currently configured to use Java 17.0.6 and Gradle 6.7.1.

Possible solution:
 - Upgrade Gradle wrapper to 7.2 version and re-import the project

### Context

I made this a feature request instead of a bug report because it can be mitigated by using Android Studio Chipmunk. But this is undesirable.

### Possible implementation

I assume it's necessary to fix the compatibility of the Gradle and Java versions, maybe with the Java version specified in .tool-versions .

**Comment by jefft0** [comment]
Will also need to update React Native at the same time.

**Comment by jefft0** [comment]
@D4ryl00 will upgrade to the latest React Native. (Hard, will need to consult with other devs.)

**Comment by jefft0** [comment]
@D4ryl00 will work on upgrading to React Native 0.73.

**Comment by jefft0** [comment]
Working on PR https://github.com/berty/berty/pull/4852 .

**Comment by JASIM0021** [comment]
Is this issue still pending?

**Comment by costinberty** [comment]
@JASIM0021 yes, it is still pending. We are working on upgrading to the latest versions of API to be again compatible with the Play Store. It is linked with this issue https://github.com/berty/berty/issues/4851

**Comment by jefft0** [comment]
PR https://github.com/berty/berty/pull/5020 is merged which updates the README build instructions for the latest Android Studio. Closing as resolved.
